build.gradle

```diff
@@ -32,7 +32,7 @@ dependencies {
 	compile group: 'org.springframework.security', name: 'spring-security-web'
 	compile group: 'org.springframework.security', name: 'spring-security-config'
 
-
+	testCompile group: 'org.mockito', name: 'mockito-all'
```

Comment:
Try to keep the gradle file organized, first the compiles and the tests in the bottom. 
(move it to line 39)